apps/extensions/src/content/panel/screens/favoriteScreen/Favorites.tsx

```
@@ -0,0 +1,232 @@
+                }}
+                onKeyDown={(e: React.KeyboardEvent) => {
+                  if (e.key === 'Enter' || e.key === ' ') {
+                    if (
+                      e.target instanceof HTMLElement &&
+                      e.target.localName !== 'ul'
+                    ) {
+                      e.preventDefault();
+                    }
+                  }
+                }}
+                role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)